packages/react-examples/src/react/ChoiceGroup/ChoiceGroup.Custom.Example.tsx

```diff
@@ -1,38 +1,28 @@
 import * as React from 'react';
 import { ChoiceGroup, IChoiceGroupOption } from '@fluentui/react/lib/ChoiceGroup';
-import { Dropdown, IDropdownStyles, IDropdownOption } from '@fluentui/react/lib/Dropdown';
 import { mergeStyles } from '@fluentui/react/lib/Styling';
+import { CatIcon } from '@fluentui/react-icons-mdl2';
 
 export const ChoiceGroupCustomExample: React.FunctionComponent = () => {
   return <ChoiceGroup defaultSelectedKey="B" options={options} label="Pick one" />;
 };
 
-const optionRootClass = mergeStyles({ display: 'flex', alignItems: 'baseline' });
-const dropdownStyles: Partial<IDropdownStyles> = {
-  root: { marginBottom: 0, marginLeft: 5 },
+const optionRootClass = mergeStyles({ display: 'flex', alignItems: 'center' });
+const iconStyles = {
+  marginBottom: 0,
+  marginLeft: 5,
 };
```

Comment:
(nit) Maybe add `gap: '5px'` to optionRootClass instead? I don't think we want to encourage using the `styles` prop in our examples.